src/teams/unify/team_invite.py

```diff
@@ -39,4 +40,9 @@ def team_invite(email, team1_id, team2_id):  # POST
 
     coll("teams").update_one({"_id": team1_id}, {"$push": {"outgoing_inv": team2_id}})
     coll("teams").update_one({"_id": team2_id}, {"$push": {"incoming_inv": team1_id}})
+    send_email(
```

Comment:
Let's comment this function call out for now, until we figure out how frequently we want to call it & what to include in the body.